README.rst

```diff
@@ -2,10 +2,10 @@
 matplotlib
 ##########
 
-matplotlib is a python 2D plotting library which produces publication
-quality figures in a variety of hardcopy formats and interactive
-environments across platforms. matplotlib can be used in python
-scripts, the python and ipython shell (ala matlab or mathematica), web
+matplotlib is a Python 2D plotting library which produces publication-quality
```

Comment:
The capitalization discussion was triggered by me for my book. The
conclusion I drew from it was to treat it as a proper noun, which is what I
did in my book. I think what was also acceptable is to treat it just simply
as a regular noun, but to stop doing lower-casing the name if it is the
first word of the sentence.

On Mon, Jun 6, 2016 at 2:13 PM, Eric Firing <notifications@github.com>
wrote:

> In README.rst
> <https://github.com/matplotlib/matplotlib/pull/6535#discussion_r65939985>:
>
> > @@ -2,10 +2,10 @@
> >  matplotlib
> >  ##########
> >
> > -matplotlib is a python 2D plotting library which produces publication
> > -quality figures in a variety of hardcopy formats and interactive
> > -environments across platforms. matplotlib can be used in python
> > -scripts, the python and ipython shell (ala matlab or mathematica), web
> > +matplotlib is a Python 2D plotting library which produces publication-quality
>
> Sounds familiar, and I don't remember the outcome, either. I would say,
> though, that the lack of capitalization at the start of a sentence looks
> terrible. Matplotlib certainly should be capitalized there, so the
> remaining question is whether it should be capitalized everywhere. I think
> it should be. I don't see any disadvantages. It looks like that is the
> practice with Python, based on a very quick look at python.org. Same with
> IPython and Jupyter. Pandas is lower case, but italicized--that only works
> with styled text, so I don't consider it a good model.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6535/files/61e288eabc4a02ce5474dc705274e1109184a61f#r65939985>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARy-PxSeVe1_qq-mEzb-NS-y-1nzgU7ks5qJGNLgaJpZM4IuTy8>
> .
>